cmd/sink/filters/filters.go

```diff
@@ -25,14 +25,17 @@ import (
 )
 
 const (
-	globalKeyEnvVar = "KUBEARCHIVE_NAMESPACE"
-	filtersCmName   = "sink-filters"
+	FiltersCmName        = "sink-filters"
+	GlobalNamespace      = "___global___"
```

Comment:
I don't think that the sink should be exporting these variables. The sink already has to import `github.com/kubearchive/kubearchive/cmd/operator/api/v1alpha1` to parse the KubeArchiveConfigResources from yaml and having the operator start importing from the sink makes it easier for us to create an infinite import loop. I think the either the operator should export them or we could put them in a shared package, like `pkg/cel`

Reply:
Fixed!